### ISSUE TITLE
setLedTxt errors if variables are used

### ISSUE THREAD

**Issue by mhantsch** [UltimateHackingKeyboard/firmware#1011]
As a follow-up from UltimateHackingKeyboard/firmware#933 ...

Error at countdown 1/4/17: Unrecognized command: count
> 4 | [setLedTxt 1000 $count](https://github.com/UltimateHackingKeyboard/firmware-uhk80/issues/new#/macro/7?actionIndex=0&lineNr=4&columnNr=17&inlineEdit=true)
>   |

(totally not urgent, but I don't want this to be forgotten)

**Comment by kareltucek** [comment]
Please retest with string interpolation `setLedTxt 1000 "$count"`. I didn't check, but I think this syntax may be illegal.

**Comment by mhantsch** [comment]
You are right, it works with the double-quotes. On the UHK60, however, it also worked without.

Checking the reference manual, it _should_ also work without the quotes:

```
COMMAND = setLedTxt <timeout, or 0 forever (INT)> { STRING | VALUE }
```

This one checks out:
```
STRING = "<interpolated string>" | '<literal string>'
```

However, `VALUE` doesn't check out, if I go the `VALUE` -> `INT` -> `VARIABLE_EXPANSION` -> `$<variable name(IDENTIFIER)` path:

```
VALUE = INT | BOOL | FLOAT
INT = PARENTHESSED_EXPRESSION | VARIABLE_EXPANSION | [0-9]+ | -[0-9]+
VARIABLE_EXPANSION = $<variable name(IDENTIFIER)> | $<config value name> | $currentAddress | $thisKeyId | $queuedKeyId.<queue index (INT)> | $keyId.KEYID_ABBREV
```

So either the implementation still has a bug, or the BNF in the reference manual is incorrect.

For the time being, I'm happy with the "put it in quotes" fix. At least that gets rid of errors popping up in Agent. You can mark this issue as low priority for now.

**Comment by kareltucek** [comment]
Indeed looks like a bug. Will look into it eventually.
